### PR TITLE
PT belt is here

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Add in the PT belt.

image

Put it on uniform so that you won't get run over by mech. Truly, innovative technology.

## Why It's Good For The Game

For people have prior service, they cannot escape the PT belt. It's really a joke RP item for people who know what the PT belt made for, so it's just fluff.

## Changelog

:cl:
add: PT belt is here
/:cl:

### CODE REVIEW

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 1]
```diff
@@ -6,6 +6,19 @@
 	name = "broken armor module"
 	desc = "You better be debugging."
 
+
+/**
+ * PT belt
+ */
+
+/obj/item/armor_module/module/pt_belt
+	name = "\improper Physical training reflective belt"
```
Comment (XSlayer300) [inline code/modules/clothing/modular_armor/attachments/modules.dm:15]:
```suggestion
	name = "\improper physical training reflective belt"
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Whats the point of this exactly you need to explain in the pr

**Comment by Moggram** [comment]
> Whats the point of this exactly you need to explain in the pr

I see some one has never been in the army.

**Comment by GreatKingWombat** [comment]
No fun allowed I guess.

**Comment by Al-1ce** [comment]
> Can the mech ran you over though, what the belt even do lad

It's like one of those yellow/neon vests police might wear. It's meant for high-visibility. Nothing else.

**Comment by novaepee** [comment]
will make it attach to uniform instead of armor lel

**Pull request merged**